### DISCUSSION TITLE
[plugin:vite:import-analysis] Failed to resolve import "/Users/wanting/vite-project/node_modules/antd/es/theme/style/index" from "src/views/Home.tsx". Does the file exist?

### DISCUSSION THREAD

**Discussion by shaojintian** [ant-design/ant-design#42156] [Q&A]
[plugin:vite:import-analysis] Failed to resolve import "/Users/wanting/vite-project/node_modules/antd/es/theme/style/index" from "src/views/Home.tsx". Does the file exist?

这是antd版本问题么

**Comment by ouyanghui100** [comment]
我也遇见了这个问题，引入theme就会报这个错误，请问你这个解决了吗？
